lib/library.go

```diff
@@ -38,15 +38,15 @@ func StartNode(configJSON *C.char) *C.char {
 		return makeJSONResponse(err)
 	}
 
-	_, err = statusAPI.StartNodeAsync(config)
-	return makeJSONResponse(err)
+	statusAPI.StartNodeAsync(config)
+	return makeJSONResponse(nil)
 }
 
 //StopNode - stop status node
 //export StopNode
 func StopNode() *C.char {
-	_, err := statusAPI.StopNodeAsync()
-	return makeJSONResponse(err)
+	statusAPI.StopNodeAsync()
```

Comment:
just a generic question: does anyone know why we call async inside "sync" methods here? @adambabik @divan @dshulyak ?

Reply:
it seems thats just an unfortunate naming, i don't see it here as a problem

Reply:
yeah, I'm just cusious